pallets/parachain-staking/src/migrations.rs

```diff
@@ -36,6 +36,35 @@ use frame_support::{
 };
 use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
 
+/// Migration to patch increase candidate bond not updating candidate pool
+pub struct UpdateCandidatePoolWithTotalCounted<T>(PhantomData<T>);
+impl<T: Config> OnRuntimeUpgrade for UpdateCandidatePoolWithTotalCounted<T> {
+	fn on_runtime_upgrade() -> Weight {
+		let (mut reads, mut writes) = (0u64, 0u64);
+		for (account, state) in <CandidateState<T>>::iter() {
+			reads += 1u64;
+			if state.is_active() {
```

Comment:
writes are much heavier than reads, so it's probably worthwhile to only update storage for affected candidates